djangocms_blog/search_indexes.py

```diff
@@ -47,32 +47,32 @@ def get_model(self):
         return Post
 
     def get_search_data(self, post, language, request):
-        optional_attributes = []
```

Comment:
It seems a zombie code, yeah. The idea behind, if i remember correctly, was to provide optional attribute to extend the search. Maybe we could just remove it and eventually develop the idea in a separated branch.